scanamo/src/main/scala/org/scanamo/generic/AutoDerivation.scala

```diff
@@ -17,16 +17,13 @@
 package org.scanamo.generic
 
 import magnolia.Magnolia
-import org.scanamo.DynamoFormat
+import org.scanamo.{ DynamoFormat, DynamoFormatDefaultInstances }
 
 import scala.language.experimental.macros
 
-trait AutoDerivation extends Derivation {
-  type Typeclass[A] = ExportedDynamoFormat[A]
+trait AutoDerivation extends Derivation with DynamoFormatDefaultInstances with LowPriority
```

Comment:
Ha, yeah this is what I was trying to avoid. I wonder why Magnolia is not picking up the instance from the `DynamoFormat` companion 🤔

Reply:
It's normal Scala, there are two places it looks for implicits:
  1. The regular scope (locals members, inherited members, imports)
  2. The implicit scope (companion objects, package objects, etc.)

in that order, so if it finds something in the regular scope it never even looks in the implicit scope.

And by importing `org.scanamo.generic.auto._` we provide an implicit in regular scope that can derive an instance for any sealed trait (including `Option`).

Reply:
Oooooh yeah, so importing the contents of the `DybamoFormat` companion into the derivation trait may fix the problem with `embed`. I’ll try that!